.github/workflows/checkAndSubmitAddonMetadata.yml

```
@@ -179,8 +179,179 @@ jobs:
       uses: peter-evans/close-issue@v3
       with:
         issue-number: ${{ inputs.issueNumber }}
+
+  createReviewComment:
+    continue-on-error: true
+    # jq for windows has issues parsing multiline strings (e.g. CRLF),
+    # use linux instead.
+    runs-on: ubuntu-latest
+    needs: [checkMetadata, getAddonFilename]
+    strategy:
+      matrix:
+        python-version: [ 3.11 ]
+    permissions:
+      contents: write
+      discussions: write
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ inputs.headRef }}
+        fetch-depth: 0
+    - name: Get add-on name and version
+      id: getAddonNameAndVersion
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const fs = require('fs')
+          const addonFilename = ${{ needs.getAddonFilename.outputs.addonFileName }}
+          const contents = fs.readFileSync(addonFilename)
+          const metadata = JSON.parse(contents)
+          const addonName = metadata.addonId
+          core.setOutput('addonName', addonName)
+          const addonVersion = addonFilename.split('/')[2].replace('.json', '')
+          core.setOutput('addonVersion', addonVersion)
+    - name: Check if discussion for this add-on exists
+      id: checkDiscussion
+      run: |
+        addonId=$(
+          echo ${{ steps.getAddonNameAndVersion.outputs.addonName }} \
+        )
+        jqCode="
+        . | .[\"$addonId\"]
+        "
+        jq -e "$jqCode" discussions.json
+        # -e only sets the exit status of jq to 0 if there is "truthy" output.
+        # If no valid result is produced, i.e. no add-on id is found, jq exits with status 4.
+        # If there is a different error, a different non-zero exit code is used.
+        exit $? # Exit with the same exit code as jq
+    - name: Get repository and category id
+      id: getRepoAndCatId
+      if: failure() && steps.checkDiscussion.outcome == 'failure'
+      uses: actions/github-script@v6
+      with:
+        script: |
+          const query = `query($owner:String!, $name:String!) {
+            repository(owner:$owner, name:$name){
+                id
+                discussionCategories(first: 1) {
+                  nodes {
+                    id
+                  }
+                }
+              }
+            }`;
+          const variables = {
```

The alignment here appears off

```suggestion
            const variables = {
```